### PR TITLE
ICU-21349 calling .usage("") should unset the existing usage

### PR DESCRIPTION
**Pull request by younies**
closes: https://github.com/icu-units/icu/issues/144

##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21349
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [x] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

File: icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java [hunk 1]
```diff
@@ -1652,6 +1652,57 @@ public void unitUsage() {
...
+        // Third: with "road" usage, then the usage unsetted by calling .usage(nul)
+        assertFormatSingle("Rounding Mode propagates: rounding up",
+                "measure-unit/length-centimeter rounding-mode-ceiling",
+                "unit/centimeter rounding-mode-ceiling",
+                NumberFormatter.with()
+                        .unit(MeasureUnit.forIdentifier("centimeter"))
+                        .usage("road")
+                        .roundingMode(RoundingMode.CEILING)
+                        .usage(null), // unset
+                new ULocale("en-US"), //
```
Comment (younies) [inline icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java:1701]: delete `//`

File: icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java [hunk 2]
```diff
@@ -1652,6 +1652,57 @@ public void unitUsage() {
...
+        assertFormatSingle("Rounding Mode propagates: rounding up",
+                "measure-unit/length-centimeter rounding-mode-ceiling",
+                "unit/centimeter rounding-mode-ceiling",
+                NumberFormatter.with()
+                        .unit(MeasureUnit.forIdentifier("centimeter"))
+                        .usage("road")
+                        .roundingMode(RoundingMode.CEILING)
+                        .usage(""), // unset
+                new ULocale("en-US"), //
+                3048,            //
```
Comment (younies) [inline icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java:1689]: delete `//`

**Review by sffc — COMMENTED** [review]
LGTM except for the small formatting fixes you noted.  Thanks!

**Review by hugovdm — APPROVED** [review]
Thanks for fixing!

**Review by hugovdm — COMMENTED** [review]
(I'll be around to approve again in the morning - agreed with the comment cleanup.)

**Review by hugovdm — CHANGES_REQUESTED** [review]
(Looks like to unset an "approve", I must explicitly "request changes".)

**Review by hugovdm — APPROVED** [review]
Consider marking PRs "ready for review" rather than leaving them as draft.

The new commit looks good:
https://github.com/unicode-org/icu/commit/65005ec3e3e0dc027e6fa546ff35484762b6b99a

(Is there any way to tell that the commit we reviewed before was https://github.com/unicode-org/icu/commit/39b915f462875fac01f3fd634d26f957d8225462 ?)

**Pull request merged**